diff -urN u-boot-2020.07-orig/arch/arm/dts/sun8i-h3-nanopi-neo.dts u-boot-2020.07/arch/arm/dts/sun8i-h3-nanopi-neo.dts
--- u-boot-2020.07-orig/arch/arm/dts/sun8i-h3-nanopi-neo.dts	2024-08-27 15:54:51.621326444 +0200
+++ u-boot-2020.07/arch/arm/dts/sun8i-h3-nanopi-neo.dts	2024-08-27 15:55:55.883215234 +0200
@@ -70,3 +70,14 @@
 &usbphy {
 	usb0_id_det-gpios = <&pio 6 12 GPIO_ACTIVE_HIGH>; /* PG12 */
 };
+
+&mmc2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&mmc2_8bit_pins>;
+	vmmc-supply = <&reg_vcc3v3>;
+	vqmmc-supply = <&reg_vcc3v3>;
+	bus-width = <8>;
+	non-removable;
+	status = "okay";
+};
+
